test/releasers/go-yoshi.ts

```diff
@@ -67,10 +72,13 @@ describe('GoYoshi', () => {
         releasePR.gh,
         'getFileContentsOnBranch'
       );
-      // CHANGELOG is not found, and will be created:
       getFileContentsStub
-        .onCall(0)
-        .rejects(Object.assign(Error('not found'), {status: 404}));
+        .withArgs('version.go', 'master')
+        .resolves(buildFileContent('version.go'));
```

Comment:
I think it might be worth adding a test case that confirms that a missing `version.go` is a `noop`.

Reply:
I can add an explicit test if you think that is needed but I showed in a couple of followup commits this is confirmed by current tests
![Screen Shot 2021-11-15 at 2 13 52 PM](https://user-images.githubusercontent.com/6331106/141857127-55835156-4046-49c1-8144-a7e05cd017f2.png)
: